src/Applications/GEOSdas_App/fvsetup

```diff
@@ -3548,7 +3549,7 @@ sub set_ldasANA {
             die ">>> Error <<< Cannot find directory, $ldashold;"
                 unless -d $ldashold;
 
-        @bkglfo_files = (<$ldashold/*bkg.lfo*${nymdb}_${hhb}*>);
+        @bkglfo_files = (<$ldashold/*bkg*lfo*${nymdb}_${hhb}*>);
```

Comment:
I noticed that in this line you changed `*bkg.lfo*` (with a dot in the middle) back to `*bkg*lfo*`.  Since we're looking for files whose names include "bkg.lfo", why can't we use the more specific string?

Reply:
there are two possible file names here , one is the tarr file  from adas archive ( bkglfo), the other one is a collection that user may provide ( bkg.lfo,  *nc4) . so * bkg*lfo* will serve for both. 
I think in the next revision to have bootstrap for ldas we will remove the option of user providing a collection. If adas ICS archive don't have bkglfo tar in rs/ ,  ldas will be in bootstrap.